src/repo/repo.service.ts

```diff
@@ -373,12 +373,12 @@ export class RepoService {
     }
 
     let repo;
-
+    const range = rangeOption?.range! || 30;
```

Comment:
This will probably work better.

```suggestion
const range = rangeOption?.range ?? 30;
```